lib/cpl.rb

```diff
@@ -243,14 +244,23 @@ def self.process_option_params(params)
     check_unknown_options!(except: @commands_with_extra_options)
     stop_on_unknown_option!
 
-    def self.validate_options!(options)
+    def self.validate_options!(options) # rubocop:disable Metrics/MethodLength
       options.each do |name, value|
-        raise "No value provided for option '#{name}'." if value.to_s.strip.empty?
+        normalized_name = ::Helpers.normalize_option_name(name)
+        raise "No value provided for option #{normalized_name}." if value.to_s.strip.empty?
```

Comment:
Prints, e.g., `--skip-secret-access-binding` instead of `skip_secret_access_binding` (internal name).